t/ea4.t

```diff
@@ -322,6 +326,31 @@ Please remove these packages before continuing the update.
     return;
 }
 
+sub test_blocker_file_behavior : Tests(8) ($self) {
+
+    my $ea_info_check = sub {
+        message_seen( 'INFO' => "Checking EasyApache profile compatibility with AlmaLinux 8." );
+    };
+
+    $self->{mock_cpev}->redefine( _get_ea4_profile => sub { $self->{mock_profile}->contents('{}'); return PROFILE_FILE } );
+
+    my $cpev = bless {}, 'cpev';
+    $cpev->{_abort_on_first_blocker} = 1;    # enforce a die
```

Comment:
note this can also be replaced by: `my $cpev = cpev(_abort_on_first_blocker => 1 )`

Reply:
applied via e43b899ec861d1d000e3779bebf37075ccd4f82b